data/projects/create_jenkins_plugin.md

```diff
@@ -6,20 +6,35 @@ If you find this fun you'll be the one knowing this component best and will
 get the chance to maintain it on your own if you want - always with the
 support of the community guarding your back of course.
 
+#### Motivation
+
+Jenkins is the most popular ci-cd server, and there are many corporations 
+who already use it. Integrating coala into Jenkins can attract business 
+clients better who could end up sponsoring/helping out coala.
+
+#### Final Goal
+
+The aim of the project is to make a plugin for Jenkins which can do 
+static analysis on code and make these visible to the user.
+
 #### Milestones
 
-##### GSOC 2017 COMMUNITY BONDING
+##### GSOC 2017 COMMUNITY BONDING(May 4 - May 30)
 
 * A test jenkins instance exists.
 * An official repository exists with rultor set up.
 
-##### GSOC 2017 MIDTERM
+##### GSOC 2017 PHASE 1(May 30 - June 30)
+
+* A parser exists for parsing coala result.
+
+##### GSOC 2017 PHASE 2(June 30 - July 28)
 
 * A working plugin with minimal functionality exists.
 * A basic testsuite exists.
```

Comment:
The testsuite should test everything that exsits at that point. As the plugin is just rudementary, the testsuite also is simple at this stage.